task5/src/main/java/com/clouway/adapter/http/LoginController.java

```diff
@@ -31,24 +34,24 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
     String password = request.getParameter("password");
 
     if (userValidator.isValid(email, password)) {
```

Comment:
`CredentialsValidator` if you pass `email` and `pass` because you pass user's credentials that want to validate in your case.
DataValidator is too common and is not clear what you validate ... what kind of data.